package.json

```diff
@@ -1,112 +1,109 @@
+		"date-picker-legacy": "webpack --config webpack.production-plugin.date-picker.legacy.js",
+		"date-picker": "webpack --config webpack.production-plugin.date-picker.js",
+		"messenger": "webpack --config webpack.production-plugin.messenger.js",
+		"messenger-legacy": "webpack --config webpack.production-plugin.messenger.legacy.js",
+		"rating-legacy": "webpack --config webpack.production-plugin.rating.legacy.js",
+		"rating": "webpack --config webpack.production-plugin.rating.js",
+		"speech-input-legacy": "webpack --config webpack.production-plugin.speech-input.legacy.js",
+		"speech-input": "webpack --config webpack.production-plugin.speech-input.js",
+		"speech-output-legacy": "webpack --config webpack.production-plugin.speech-output.legacy.js",
+		"speech-output": "webpack --config webpack.production-plugin.speech-output.js",
+		"webchat-legacy": "webpack --config webpack.production.legacy.js",
+		"webchat": "webpack --config webpack.production.js",
+		"cypress:open": "cypress open",
+		"cypress:serve": "http-server -a localhost -p 8787 dist/",
+		"cypress:run": "cypress run",
+		"test:cypress": "run-p -r cypress:serve cypress:run",
+		"test:es5-legacy": "es-check es5 ./dist/*.legacy.js",
+		"test": "npm run test:es5-legacy && npm run test:cypress",
+		"pretest": "npm run bundle"
+	},
+	"dependencies": {
+		"@braintree/sanitize-url": "^5.0.2",
+		"@cognigy/socket-client": "^4.5.4",
+		"@emotion/cache": "^10.0.29",
+		"@emotion/react": "^11.7.1",
+		"@emotion/styled": "^11.6.0",
+		"dompurify": "^2.0.7",
+		"flatpickr": "4.6.3",
+		"immutable": "^4.0.0-rc.12",
+		"lodash": "^4.17.21",
+		"moment": "^2.24.0",
+		"path-parse": "^1.0.7",
+		"react": "^17.0.2",
+		"react-animate-height": "^2.0.23",
+		"react-dom": "^17.0.2",
+		"react-player": "^2.9.0",
+		"react-redux": "^7.2.6",
+		"react-responsive-carousel": "^3.2.22",
+		"react-textarea-autosize": "^8.3.3",
+		"redux": "^4.0.4",
+		"styled-jss": "^2.2.3",
+		"stylis-rtl": "^1.0.1",
+		"tinycolor2": "^1.4.2",
+		"uifx": "^2.0.7"
+	},
+	"devDependencies": {
+		"@babel/cli": "^7.16.7",
+		"@babel/core": "^7.16.7",
+		"@babel/plugin-proposal-class-properties": "^7.16.7",
+		"@babel/plugin-proposal-nullish-coalescing-operator": "^7.16.7",
+		"@babel/plugin-proposal-object-rest-spread": "^7.16.7",
+		"@babel/plugin-proposal-optional-chaining": "^7.16.7",
+		"@babel/plugin-syntax-dynamic-import": "^7.8.3",
+		"@babel/plugin-transform-object-assign": "^7.16.7",
+		"@babel/plugin-transform-runtime": "^7.16.7",
+		"@babel/preset-env": "^7.16.7",
+		"@babel/preset-react": "^7.16.7",
+		"@babel/preset-typescript": "^7.16.7",
+		"@babel/runtime": "^7.16.7",
+		"@types/dompurify": "^2.0.0",
+		"@types/react": "^16.9.11",
+		"@types/react-dom": "^16.9.4",
+		"@types/react-redux": "^6.0.12",
+		"@types/tinycolor2": "^1.4.3",
+		"@types/uuid": "^3.4.6",
+		"@types/whatwg-fetch": "0.0.33",
+		"babel-loader": "^8.2.3",
+		"babel-polyfill": "^6.26.0",
+		"buffer": "^6.0.3",
```

Comment:
why was this package added?

Reply:
I was able to build and test successfully without it

Reply:
True, seems to be a leftover from testing, works fine without, removed it